EIPS/eip-7933.md

```diff
@@ -0,0 +1,152 @@
+- `chain_id`: optional chain identifier (e.g. `1` for Ethereum Mainnet, `324` for ZkSync Era)
+- URL parameters: key-value pairs representing the parameters of the RPC method
+
+### Optional components
+
+- `requests_b64`: for `multi-request`, a base64-encoded array of RPC calls
+
+### Example URIs
+
+**eth_sendTransaction**
+
+```
+ethereum:eth_sendTransaction-0x0000000000000000000000000000000000000000@324?from=CURRENT_ACCOUNT&to=0x1c7D4B196Cb0C7B01d743Fbc6116a902379C7238&gas=0x76c0&gasPrice=0x4a817c800&value=0x0&data=0xa9059cbb000000000000000000000000ffce4d191cb5007ee9ad7226581f7e217b68cafe00000000000000000000000000000000000000000000000000000000000f4240636166656369746f
+```
+
+**wallet_addEthereumChain**
+
+`
+ethereum:wallet_addEthereumChain-0x0000000000000000000000000000000000000000?chainId=0x64&chainName=Gnosis&rpcUrls[0]=https%3A%2F%2Frpc.gnosischain.com&iconUrls[0]=https%3A%2F%2Fxdaichain.com%2Ffake%2Fexample%2Furl%2Fxdai.svg%2Chttps%3A%2F%2Fxdaichain.com%2Ffake%2Fexample%2Furl%2Fxdai.png&nativeCurrency[name]=xDAI&nativeCurrency[symbol]=xDAI&nativeCurrency[decimals]=18&blockExplorerUrls[0]=https%3A%2F%2Fgnosisscan.io
+`
+
+**multiRequest**
+
+```
+ethereum:multiRequest-0x0000000000000000000000000000000000000000@324?requests_b64=W3sibWV0aG9kIjoid2FsbGV0X3N3aXRjaEV0aGVyZXVtQ2hhaW4iLCJwYXJhbXMiOlt7ImNoYWluSWQiOiIweDE0NCJ9XX0seyJtZXRob2QiOiJldGhfc2VuZFRyYW5zYWN0aW9uIiwicGFyYW1zIjpbeyJmcm9tIjoiMHg4MDgyZGE2NzcxMGMxNGU3ZjY2OGVmYzczYWMyN2FkNmIyZDdjYWZlIiwidG8iOiIweDFjN0Q0QjE5NkNiMEM3QjAxZDc0M0ZiYzYxMTZhOTAyMzc5QzcyMzgiLCJnYXMiOiIweDc2YzAiLCJnYXNQcmljZSI6IjB4NGE4MTdjODAwIiwidmFsdWUiOiIweDAiLCJkYXRhIjoiMHhhOTA1OWNiYjAwMDAwMDAwMDAwMDAwMDAwMDBmZmNlNGQxOTFjYjUwMDdlZTlhZDcyMjY1ODFmN2UyMTdiNjhjYWZlMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwZjQyNDA2MzYxNmY2NjYzNjk3In1dfV0=
+```
+
+## Rationale
+
+- **Intents** allow wallet interfaces to preemptively guide users through transactions with clear context.
+- **Chain-specific targeting** improves safety and predictability across networks.
+- **Multi-request** support allows for batch operations, simplifying advanced workflows (e.g., approval + vault deposit).
+- **QR-friendly design** enables real-world usage (e.g., payments, POS, offline signing).
+
+Alternate designs such as JSON files or app links introduce complexity and dependency on frontend apps, while EI-URI stays fully compatible with existing URI schemes and QR infrastructure.
+
+## Backwards Compatibility
+
+The proposed format extends the existing `ethereum:` URI standard but remains backward-compatible. Wallets that do not support the extended format will simply ignore unknown parameters.
+
+## Test Cases
+
+TBD – Will include example URIs and expected behavior in compatible wallets.
+
+## Reference Implementation
+
+A reference implementation could be built as a lightweight URI parser in JavaScript that extracts and maps the URL to a `window.ethereum.request(...)` call.
```

Comment:
`could be build` is not  correct  for this section, either add reference implementation link/details or mark it as <--TODO-->

Reply:
ty